docs/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Ipheco Chris Blog</title>
+    </head>
+    <body>
+        <header>
+            <h1>Ipheco Chris Blog</h1>
+            <nav>
+                <a href="#">Home</a>
+                <a href="#">About Me</a>
+                <a href="#">Contact Us</a>
+            </nav>
+        </header>
+
+        <section>
+            <h1>Meet Ipheco Chris</h1>
+            <img src="./images/Ipheco.jpeg" alt="Ipheco" />
+
+            <p>
+                Anyaefiena Ifechukwu Mathias alias Ipheco Chris is a
+                well-known figure in the world of cryptocurrency trading. He
+                is known for his interest, expertise and skills in
+                blockchain technology and digital currencies. With extensive
+                experience in crypto trading, he has become a trusted
+                advisor to many investors in the field.
+            </p>
+
+            <p>
```

Comment:
If you want create a line break, use the `<br/>` element instead of using another paragraph element. What you did is perfectly, I'm just pointing out a more intuitive way of doing it.

Reply:
Alright, I have made the Changes sir, can you kindly recheck this and give me go ahead to merge?

Reply:
@promise-J
@ChidiPatrick
@OgJaykay
@Pieta9221
@MiniHack
@Melumm
I have made the Changes to my Lesson 1 & 2 Task 1, Please Kindly Check it out.